docs/components/tracking_script.rst

```diff
@@ -98,25 +98,30 @@ JavaScript Form processing hooks
        }
    };
 
-If you wish to run additional code before or after submission of the Form, create a ``MauticFormCallback`` object.
-In the example code, replace ``replaceWithFormName`` with the name of your Form. 
+If you wish to run additional code before or after submission of the form, create a ``MauticFormCallback`` object.
```

Comment:
```suggestion
If you wish to run additional code before or after submission of the Form, create a ``MauticFormCallback`` object.
```

Please capitalise Form - per the Google StyleGuide definitions we capitalise all the bundles/components of Mautic. Vale should flag these up as a warning.